osu.Game.Tests/Beatmaps/IO/LegacyBeatmapExporterTest.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.IO;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions;
+using osu.Framework.Extensions.ObjectExtensions;
+using osu.Framework.Testing;
+using osu.Game.Beatmaps;
+using osu.Game.Database;
+using osu.Game.Tests.Resources;
+using osu.Game.Tests.Visual;
+using MemoryStream = System.IO.MemoryStream;
+
+namespace osu.Game.Tests.Beatmaps.IO
+{
+    [HeadlessTest]
+    public partial class LegacyBeatmapExporterTest : OsuTestScene
```

Comment:
Just wondering if this test would be better suited to the `LegacyBeatmapEncoderTest` class?

Reply:
I'd actually prefer them to be here. Encoder shouldn't be caring about backcompat logic, encoder should be encoding, i.e. dumb formatting logic. This is where I'd also put legacy export stability tests that you asked for in relation to BSS and which we don't currently have.